online/src/main/scala/ai/chronon/online/FetcherBase.scala

```diff
@@ -309,14 +305,85 @@ class FetcherBase(kvStore: KVStore,
                   ex.printStackTrace()
                   throw ex
               }
+              logger.info(s"[test] groupByServingInfo ${groupByServingInfo}")
```

Comment:
assuming that these log statements will be removed before merging

Reply:
we can change the log level to logger.debug

Reply:
Yep these logs will be removed before merging, just use them for debugging